manifests/secrets.pp

```diff
@@ -1,6 +1,8 @@
-# Configure a strongSwan secrets configuration.
+# StrongSwan secrets configuration.
+# **$selectors** - list of selectors for current secret(@see ipsec.secrets(5))
 define strongswan::secrets(
-  $options = {},
+  $selectors = [$title],
```

Comment:
please add a datatype for the new parameter

Reply:
Added datatype to parameters. Actually the whole module doesn't contain datatypes specified and this can be a bigger topic for reviewing.

Reply:
thanks. We require datatypes for all new/changes parameters. Of course dedicated PRs that just add types are always most welcome :)